dcpy/library/cli.py

```diff
@@ -36,18 +36,19 @@ def archive(
         typer.echo(message)
     else:
         a = Archive()
-        a(
-            path=path,
-            output_format=output_format,
-            push=push,
-            clean=clean,
-            latest=latest,
-            name=name,
-            compress=compress,
-            inplace=inplace,
-            postgres_url=postgres_url,
-            version=version,
-        )
+        for output_format in output_formats:
```

Comment:
A little silly to be doing it this far upstream in the process. But that's what we do now when we specify multiple formats. And since we're going to move away from multiple formats in general, doesn't seem worth the effort to produce multiple output files from the same gdal object (especially since data library is a maze. Initially thought I'd just quickly do that but it's not as simple as one might hope)

Reply:
And I'm still preserving this functionality rather than going just to pgdump for these cases 
1. just to not change functionality at the moment
2. for datasets where we historically always have csv outputs, converting those to parquet would be easier than with the pgdumps. Not sure if we will want to for historical recipe datasets but just in case.